### PR TITLE
Tangential THICKMATERIAL-related fixes

### LINKED ISSUE #9460: Mech Syringe gun ignores hardsuits/thick clothing
What it says on the tin. Not sure if intended ?

### PR DESCRIPTION
**Pull request by Cheridan**
-Monkey bites now respect THICKMATERIAL. Essential for monkey mode best mode.
-Fixes Combat Hypo for Nuke Ops (wtf were you thinking goofball??), important since their thick hardsuits prevent many other forms of healing. Now it'll be better at healing and very good at helping up ops who are in crit.

fixes #9460

### CODE REVIEW

**Comment by optimumtact** [comment]
Fixes #8996 ?

**Comment by Iamgoofball** [comment]
I think the combat hypo was pudl when he was doing the stimulant shit

Either that or I fucked up a find/replace during the switch to goonchem

**Comment by Cheridan** [comment]
@optimumtact It helps with #8996, I wouldn't say it fixes it. THICKMATERIAL still blocks packs and ointment (which its supposed to). We could replace the starter kit they get with another one that has patches instead.

**Comment by AnturK** [comment]
This is something that came up recently that i'm not sure is intended and is related : mech syrgine gun ignores the thick flag.

**Comment by Cheridan** [comment]
Sure, I'll update this soon.

**Comment by Cheridan** [comment]
done

goddamn mech syringe code makes me want to :man: :gun:

File: code/game/mecha/equipment/tools/medical_tools.dm [hunk 1]
```diff
@@ -306,47 +306,48 @@
...
-	reagents.trans_to(S, min(S.volume, reagents.total_volume))
-	syringes -= S
-	S.icon = 'icons/obj/chemical.dmi'
-	S.icon_state = "syringeproj"
+	var/obj/item/weapon/reagent_containers/syringe/mechsyringe = syringes[1]
+	mechsyringe.forceMove(get_turf(chassis))
+	reagents.trans_to(mechsyringe, min(mechsyringe.volume, reagents.total_volume))
+	syringes -= mechsyringe
+	mechsyringe.icon = 'icons/obj/chemical.dmi'
+	mechsyringe.icon_state = "syringeproj"
```
Comment (Miauw62) [inline code/game/mecha/equipment/tools/medical_tools.dm]: lol, mech code

**Pull request merged**